test/ui-test/src/drivers/SquishDriverVerification.py

```diff
@@ -20,15 +20,15 @@ def verify_screen(objName: str, timeout: int=_MAX_WAIT_OBJ_TIMEOUT):
 
 def verify_object_enabled(objName: str, timeout: int=_MIN_WAIT_OBJ_TIMEOUT, condition: bool=True):
     result = is_loaded_visible_and_enabled(objName, timeout)
-    test.verify(result[0] == condition, "object not enabled")
+    test.verify(result[0] == condition, "Checking if object enabled")
```

Comment:
Isn't the message parameter what is shown when the verification fails? Maybe it should be
```js
"We wanted the object to be " + (condition ? "enabled" : "disabled") + ", but it was not"
```
(this code is not valid python)

If I'm not mistaken, you might want to do that to all the others below too

Reply:
I think we've been a bit confused with that, the message is displayed whether the test passes or fails, which is why I rewrote it. See squish information here: https://doc.qt.io/squish/squish-api.html#test-verify-function

Reply:
For simple comparisons test.compare() might be a better option because it will also output the difference between actual and expected.

Reply:
Yeah! I agree! Here I was just updating wrong texts since there was the confusion that they were only displayed when failure but is not the case!